src/plugins/janus_streaming.c

```diff
@@ -10192,6 +10257,13 @@ static void janus_streaming_relay_rtp_packet(gpointer data, gpointer user_data)
 					rtp.extensions.min_delay = s->min_delay;
 					rtp.extensions.max_delay = s->max_delay;
 				}
+				if(session->abscapturetime_ext && (session->abscapturetime_src_ext_id > 0 && session->abscapturetime_src_ext_id < 15)) {
+					uint64_t abs_ts = 0;
+					if(janus_rtp_header_extension_parse_abs_capture_time((char *)packet->data, packet->length,
+									session->abscapturetime_src_ext_id, &abs_ts) == 0) {
```

Comment:
Per code style, there's too much indentation here: it should be just one indent deeper than the line that follows.

Reply:
fixed